NARFO_BE/NARFO_BE/encryption.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NARFO_BE
+{
+    public class encryption
+    {/*
+        public static String sha256_hash(string value)
+        {
+            StringBuilder Password = new StringBuilder();
+
+            using (var hash = SHA256.Create())
+            {
+                Encoding enc = Encoding.UTF8;
+                Byte[] result = hash.ComputeHash(enc.GetBytes(value));
+
+                foreach (Byte b in result)
+                    Password.Append(b.ToString("x2"));
+            }
+
+            return Password.ToString();
+        }*/
+        
+
+
+       public static byte[] SaltBytes { get; private set; }
+        public enum Supported_HA
+        {
+            SHA256
+        }
+
+       
+        public static string ComputeHash(string plainText)
+        {
+            string value = "shasha";
+            byte[] salt = Encoding.ASCII.GetBytes(value);
+            //int minSaltLength = 4, maxSaltLength = 16;
```

Comment:
A const salt is bad.

Reply:
what  do you prefer??  thank you for your view ( :